src/vos/evtree.c

```diff
@@ -2179,16 +2179,19 @@ evt_insert(daos_handle_t toh, const struct evt_entry_in *entry,
 			memcpy(&ent_cpy, entry, sizeof(*entry));
 			entryp = &ent_cpy;
 			/** We need to edit the existing extent */
-			if (ent->en_ext.ex_lo < ent_cpy.ei_rect.rc_ex.ex_lo)
-				ent_cpy.ei_rect.rc_ex.ex_lo = ent->en_ext.ex_lo;
-			if (ent->en_ext.ex_hi > ent_cpy.ei_rect.rc_ex.ex_hi)
-				ent_cpy.ei_rect.rc_ex.ex_hi = ent->en_ext.ex_hi;
-
-			/** Remove the existing node */
-			rc = evt_node_delete(tcx);
```

Comment:
minor: since we don't do in-place deletion now, the comment above "we can delete the old record and insert a merged one" could be adjusted.

Reply:
Since it appears to have failed due to DAOS-8231, I've fixed it.